records_mover/session.py

```diff
@@ -261,6 +273,15 @@ def db_driver(self, db: Union['Engine', 'Connection']) -> 'DBDriver':
             except NotImplementedError:
                 logger.debug('boto3 not installed', exc_info=True)
 
+        scratch_gcs_url = self.creds.default_scratch_gcs_url()
+        if scratch_gcs_url is not None:
+            gcs_temp_base_loc = self.directory_url(scratch_gcs_url)
+            kwargs['gcs_temp_base_loc'] = gcs_temp_base_loc
+            # TODO: will this blow up with NotImplementedError if
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker